docs/sphinx/manual/building_hpx.rst

```diff
@@ -183,6 +183,60 @@ more recent standards can be opted into explicitly. If you want to force |hpx|
 to use a specific C++ standard version, you can use the ``HPX_WITH_CXX_STANDARD``
 CMake variable; e.g. ``HPX_WITH_CXX_STANDARD=20``.
```

Comment:
As discussed in the meeting, let's move this to the list of options below. Just adding this comment as a reminder.